cleverhans/experimental/certification/optimization.py

```diff
@@ -282,6 +326,20 @@ def run_one_step(self, eig_init_vec_val, eig_num_iter_val, smooth_val,
                                 str(self.current_step) + '.json')
         with tf.gfile.Open(filename) as file_f:
           file_f.write(stats)
+
+    # Project onto feasible set of dual variables
+    if self.current_step % self.params['projection_steps'] == 0 and self.current_unconstrained_objective < 0:
+      nu = self.sess.run(self.dual_object.nu)
+      feed_dict = {
+          self.projected_dual_object.nu: nu,
+          self.dual_object.h_min_vec_ph: self.dual_object.h_min_vec_estimate,
+          self.projected_dual_object.m_min_vec_ph: self.projected_dual_object.m_min_vec_estimate
+      }
+      _, min_eig_val_h_lz = self.dual_object.get_lanczos_eig(compute_m=False, feed_dict=feed_dict)
+      feed_dict.update({self.min_eig_val_h: min_eig_val_h_lz})
```

Comment:
I think this class does not have `min_eig_val_h` member, it's declared in `projected_dual_object`